beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -211,18 +211,16 @@ func TestProcessRegistryUpdates_EligibleToActivate(t *testing.T) {
 	}
 }
 
+// EIP 7514 is Removed
 func TestProcessRegistryUpdates_EligibleToActivate_Cancun(t *testing.T) {
```

Comment:
This test is added for Deneb(PR #12959, [commit](https://github.com/prysmaticlabs/prysm/pull/12959/commits/79fcc6eb66b68915434a87cbd6f667dd69286155)). Since `MaxPerEpochActivationChurnLimit` is deleted, this test isn't necessary IMO.